### ISSUE TITLE
Content accessibility issues

### ISSUE THREAD

**Issue by brentryanjohnson** [GSA/smartpay-training#319]
Additional accessibility issues as a result of merging #312 

## Lesson 1: Purchase Program Overview
[`preview/gsa/smartpay-training/staging/training_purchase/lesson01/`](https://federalist-2e11f2c8-970f-44f5-acc8-b47ef6c741ad.sites.pages.cloud.gov/preview/gsa/smartpay-training/staging/training_purchase/lesson01/)

#### Found

Alt text insufficient for alternative description to visual image

#### Element

```
<img src="/preview/gsa/smartpay-training/staging/images/red-card.png" alt="GSA SmartPay card design" width="160" class="margin-y-2">

<img src="/preview/gsa/smartpay-training/staging/images/yellow-card.png" alt="GSA SmartPay card design" width="160" class="margin-y-2">
```

> [!NOTE]
> This `alt` text doesn't distinguish between the two images, and thus insufficient as text alternatives to the visual images. The `alt` text should describe the image in sufficient detail to serve as an alternative to someone who cannot see the image.

## Lesson 3: Preparing to Purchase
[`preview/gsa/smartpay-training/staging/training_purchase/lesson03/`](https://federalist-2e11f2c8-970f-44f5-acc8-b47ef6c741ad.sites.pages.cloud.gov/preview/gsa/smartpay-training/staging/training_purchase/lesson03/)

#### Found

> This A element has an accessible name of 'gsa smartpay website' that is the same as another A element but doesn't go to the same HREF.

#### Element

`<a class="usa-link" href="www.smartpay.gsa.gov">GSA SmartPay website</a>`

#### Found

> This A element has an accessible name of 'gsa smartpay website' that is the same as another A element but doesn't go to the same HREF.

#### Element

`<a class="usa-link" href="https://smartpay.gsa.gov/content/state-tax-information">GSA SmartPay website</a>`

#### Found

> This A element has an accessible name of 'gsa smartpay website' that is the same as another A element but doesn't go to the same HREF.

#### Element

`<a class="usa-link" href="https://smartpay.gsa.gov/content/bank-customer-service-centers">GSA SmartPay website</a>`

> [!NOTE]
> AMP flagged these three links with identical text but different link _destinations_. That is, the link text is `GSA SmartPay website`, but the page links go to three different pages. Since assistive technology users can tab through just the links on the page — without the surrounding text — the link text should be descriptive and sufficient on its own to indicate to which specific page the link with take them.

#### Example `alt` text for cards
These descriptions attempt to provide a comparable, equivalent experience for assistive technology users, by describing the images in detail. Assistive technology users can stop or skip the description if it's too long, but they have the option of accessing relatively comparable information to the visual user.

**Purchase**
> Red charge card with the word Purchase and numbers 1234 5678 9012 3456 and the name John Smith, with a bird in the background

**Travel**
> Blue charge card with the word Travel and numbers 1234 5678 9012 3456 and the name John Smith, with an airplane in the background

**Tax Advantage**
> Silver charge card with the words Tax advantage and numbers 1234 5678 9012 3456 and the name John Smith, with a car and buildings in the background

**Fleet**
> Green charge card with the word Fleet and numbers 1234 5678 9012 3456 and the name John Smith, with a road in the background

**Integrated**
> Yellow charge card with the word Intergrated and numbers 1234 5678 9012 3456 and the name John Smith, with a globe outline in the background

**Comment by weiwang-gsa** [comment]
@rebekahperillo seems "GSA SmartPay website" in purchase training lesson 3 are pointing to 3 different links,  which is causing AMP complaints, are you ok if I make the update below?  Or you can let me know what 2 and 3 text should be, thanks!

1. [GSA SmartPay website](https://smartpay.gsa.gov/)
2. [GSA SmartPay website State Tax Information](https://smartpay.gsa.gov/content/state-tax-information) 
3. [GSA SmartPay website Bank Customer Service Centers](https://smartpay.gsa.gov/content/bank-customer-service-centers)

**Comment by weiwang-gsa** [comment]
@rebekahperillo thank you, I will make the change accordingly, by the way, does user know "program website" refer to "GSA Smartpay website"?

**Comment by rebekahperillo** [comment]
Good point. Will this work?

Be sure to visit the [State Tax Information page](https://smartpay.gsa.gov/content/state-tax-information) on the GSA SmartPay website to find out more about requirements in your state.
Reach out to your A/OPC or contractor bank immediately if your account is declined. The contractor banks’ customer service phone numbers can be found on the [Contact page](https://smartpay.gsa.gov/content/bank-customer-service-centers) on the GSA SmartPay website and are listed on the back of the charge card.

**Comment by rebekahperillo** [comment]
Sounds good! :) Thank you!

Rebekah Knouse Perillo (she/her)

Lead Business Management Specialist

Center for Charge Card Management
Office of Professional Services and Human Capital Categories
Federal Acquisition Service, U.S. General Services Administration

301-938-5257

***@***.***


Follow the GSA SmartPay program on social media...Tweet with us on Twitter
<https://twitter.com/GSASmartPayNews> and friend us on Facebook
<https://www.facebook.com/GSASmartPayNews/>!
[image: image.png]

**Comment by weiwang-gsa** [comment]
@brentryanjohnson this is ready for you to review here: https://federalist-2e11f2c8-970f-44f5-acc8-b47ef6c741ad.sites.pages.cloud.gov/preview/gsa/smartpay-training/staging/

**Comment by JessicaMarine1** [comment]
@A-L33 @rebekahperillo this is ready for your review.

**Comment by rebekahperillo** [comment]
Moving to DONE. Thank you!
